scram/templates/route_manager/home.html

```diff
@@ -23,7 +23,8 @@ <h1 class="display-3 text-center text-white font-weight-bold">SCRAM</h1>
             {% endif %}
           {% endfor %}
         </select>
-        <input class="form-control mb-2" name="comment" placeholder="comment" type="text">
+        <input class="form-control mb-2 mr-2" name="comment" placeholder="comment" type="text">
+        <input class="form-control mb-2 mr-2" name="expiration" type="datetime-local">
```

Comment:
Do you want a placeholder to show the expected format?

Reply:
The date time field widget already walks you through what you need and translates it into the proper format.